### ISSUE TITLE
Error with nestjs after upgrading clerk-sdk-node to 4.12.0

### ISSUE THREAD

**Issue by larryro** [clerkinc/javascript#1513]
- [x] Review the documentation: https://clerk.com/docs
- [x] Search for existing issues: https://github.com/clerkinc/javascript/issues
- [x] Go through package changelog files.
- [x] Provide the Frontend API key from your application dashboard.

## Package + Version

- [ 4.12.0] `@clerk/clerk-sdk-node`

### Dependencies + versions

Provide a json with the dependencies used in your project (copy paste from yarn.lock / package-lock.json) or a github project / template that reproduces the issue.

**Include the @clerk/* packages and their versions!**

Example:

```json
{  
  "dependencies": {
    "@clerk/clerk-sdk-node": "4.12.0",
    "@clerk/nextjs": "4.23.0",
  },
  "devDependencies": {
  },
  "peerDependencies": {
  }
}

```

### Browser/OS
Node v18.16.1
If applicable e.g. `Chrome {version}`, `Node {version}` , `Bun {version}` , `Cloudflare worker {version}`

## Description

Describe your issue in detail, a demo repository or a sandbox would be great.
You can get the detail from here:    
 
https://github.com/clerkinc/javascript/pull/1492#discussion_r1268268560

**Comment by danieldunderfelt** [comment]
I am also encountering this problem with a trpc + expo project. 4.11.1 works.

**Comment by dimkl** [comment]
Hello @larryro 
thank you for the feedback we have released the `@clerk/clerk-sdk-node@4.12.1` which should resolve the issue.
Feel free to re-open this if it's not fixed.

**Comment by larryro** [comment]
> Hello @larryro thank you for the feedback we have released the `@clerk/clerk-sdk-node@4.12.1` which should resolve the issue. Feel free to re-open this if it's not fixed.

Thank you very much, you are very efficient.

**Comment by clerk-cookie** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
